app/views/groups/members/index.html.erb

```diff
@@ -3,7 +3,7 @@
   <%= component.with_buttons do %>
     <% if allowed_to?(:create_member?, @namespace) %>
       <%= link_to t(:".add"),
-                  new_group_member_path(@namespace),
+                  new_group_member_path(@namespace, **request.query_parameters),
```

Comment:
**nitpick (blocking):** Same comment as above, if this is to maintain tab navigation can it be updated to only include that param

Reply:
Actually instead of that if you just assign @tab in the controller you can use that in the views.

Reply:
Updated in  9d14e974555ff0bbe5c4b392f2e0b47b8a2b1865